src/buffer/channel.rs

```diff
@@ -64,7 +63,7 @@ pub fn view<'a>(
     });
 
     // TODO: Maybe we should show it to the right instead of left.
```

Comment:
Not using position in `User` position user sidebar. Will add that later. We have the information now, which was the goal.